OONIProbeUnitTests/NetworkTest.m

```diff
@@ -0,0 +1,71 @@
+//
+//  NetworkTest.m
+//  OONIProbeUnitTests
+//
+//  Created by Lorenzo Primiterra on 01/03/2019.
+//  Copyright © 2019 OONI. All rights reserved.
```

Comment:
Is there a way for us to change the default code header generate to include the standard OONI license template: https://github.com/ooni/license/blob/master/software/LICENSE.md?

Reply:
In a previous conversation we decided to remove headers at all from the source files.

Reply:
Ah yes that would also work